code/components/voip-mumble/src/MumbleAudioOutput.cpp

```diff
@@ -1331,6 +1326,24 @@ void MumbleAudioOutput::ThreadFunc()
 	CoInitialize(nullptr);
 
 	InitializeAudioDevice();
+
+	while (true) {
+		const int opusFrameTime = 40;
```

Comment:
I don't think polling more frequently in XA2 mode would harm too much, and it may lead to less dropouts in fact.